package.json

```diff
@@ -68,6 +68,7 @@
     "coveralls": "~3.0.0",
     "fs-extra": "~4.0.2",
     "istanbul": "~0.4.5",
+    "jshint": "~2.9.5",
```

Comment:
This seems redundant.

Reply:
Hi @XhmikosR 

I added `jshint` to `devDependencies` because it is required to run `npm run jshint`

is this something that you would like to see installed globally instead of project specific?

Reply:
No, I meant it's unrelated to the other changes :)

Reply:
good point - I rolled back this update here and added to a new pr https://github.com/svg/svgo/pull/1008